docs/contributing/contributing-to-dataline/templates/source-doc-template.md

```diff
@@ -0,0 +1,50 @@
+# Source Name
+
+## Overview 
+
+### Sync Overview
+#### Resulting schema
+Is the schema changed or altered in any way? If so, mention how. 
```

Comment:
Can you give an example, I don't really know what this means.

Reply:
I was thinking of if a source does any weird denesting or appends columns, that this should be mentioned here. How about this edit? 

```
Is the output schema fixed (e.g: for an API like Stripe)? If yes, point to the schema or include it here directly. Otherwise if the schema is dynamic (e.g: Postgres), describe how the source schema is mapped to Dataline concepts. An example description might be: "MagicDB tables become Dataline Streams and MagicDB columns are become Dataline Fields. In addition, an `extracted_at` column is appended to each row being read."
```

Reply:
I was thinking of if a source does any weird denesting or appends columns, that this should be mentioned here. How about this edit? 


> Is the output schema fixed (e.g: for an API like Stripe)? If yes, point to the schema or include it here directly. Otherwise if the schema is dynamic (e.g: Postgres), describe how the source schema is mapped to Dataline concepts. An example description might be: "MagicDB tables become Dataline Streams and MagicDB columns are become Dataline Fields. In addition, an `extracted_at` column is appended to each row being read."

Reply:
resolved offline